Library/Homebrew/extend/os/linux/keg_relocate.rb

```diff
@@ -0,0 +1,76 @@
+class Keg
+  def relocate_dynamic_linkage(relocation)
+    # Patching patchelf using itself fails with "Text file busy" or SIGBUS.
+    return if name == "patchelf"
+    elf_files.each do |file|
+      file.ensure_writable do
+        change_rpath(file, relocation.old_prefix, relocation.new_prefix)
+      end
+    end
+  end
+
+  def change_rpath(file, old_prefix, new_prefix)
+    return unless file.elf? && file.dynamic_elf?
+
+    begin
+      patchelf = Formula["patchelf"].bin/"patchelf"
+    rescue FormulaUnavailableError
+      return
+    end
+
+    cmd_rpath = "#{patchelf} --print-rpath '#{file}' 2>&1"
```

Comment:
Does this have to use `2>&1`? It'd be nicer to separate the parameters given you're passing in variables here.

Reply:
Agreed.